src/v2/Apps/Partner/Components/Overview/ArtistsRail.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react"
+import { Box, Flex, Text } from "@artsy/palette"
+import { createFragmentContainer, graphql } from "react-relay"
+import { ArtistsRail_partner } from "v2/__generated__/ArtistsRail_partner.graphql"
+import { RouterLink } from "v2/Artsy/Router/RouterLink"
+import {
+  PartnerArtistsCarouselRenderer,
+  PartnerArtistsPaginationContainer,
+} from "../PartnerArtists"
+
+interface ArtistsRailProps {
+  partner: ArtistsRail_partner
+}
+
+const ArtistsRail: React.FC<ArtistsRailProps> = ({ partner }) => {
+  const { slug, profileArtistsLayout } = partner
+
+  return (
+    <Box mt={4} mb={[4, 80]}>
```

Comment:
Just a general note; not a blocker or anything: I find it much easier to edit/understand when no child components contain vertical margins of any kind. Typically I'll either extend BoxProps and spread `...rest` props; or just handle with `<Box>`es in the outer context and keep all margins at the highest level (typically the app/page level). Just makes it much easier to edit once you've got a lot of things going on in the page. Nothing wrong with this since these are all one-offs — personal preference.

Reply:
Got it i will stick to your option in the future. Also, I plan to make some refactoring of the artist tab after all related branches will be merged I will fix such places.